proxy/envoy/testdata/lds-v0-egress-rule-auth-optin.json.golden

```diff
@@ -0,0 +1,658 @@
+{
+  "listeners": [
+   {
```

Comment:
kill. We don't need to update all golden files. You just need two files, to showcase optin and optout.

Reply:
Done. I add separated test for optin and optout so just need 2 golden files now.